src/type/directives.js

```diff
@@ -84,10 +86,13 @@ export class GraphQLDirective {
           name: argName,
           description: arg.description === undefined ? null : arg.description,
           type: arg.type,
-          defaultValue: arg.defaultValue
+          defaultValue: arg.defaultValue,
+          astNode: arg.astNode || null,
```

Comment:
It would be nice if this was `undefined` instead of `null` since the two mean slightly different things. I think that can be achieved by just removing this `|| null`